projects/rero/ng-core/src/lib/record/editor/editor.component.scss

```diff
@@ -14,9 +14,9 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-@import "node_modules/bootstrap/scss/functions";
-@import "node_modules/bootstrap/scss/mixins";
-@import "node_modules/bootstrap/scss/variables";
+@import "~bootstrap/scss/functions";
+@import "~bootstrap/scss/mixins";
+@import "~bootstrap/scss/variables";
```

Comment:
`tild` imports seems depreciated : https://stackoverflow.com/a/70136354/5595377 (containing links to more official doc)

Additionally, in my broken ng-core version (bootstrap 4.6.2), `functions`, `mixins` and `variables` are not longer available. It was replaced by `_functions`, ... but all these files are includes into `@import node_modules/bootstrap/scss/bootstrap`

![image](https://user-images.githubusercontent.com/10031585/214246422-f942fe53-68f2-4e7b-98db-fda539027bb9.png)